File: java/src/test/java/com/showmeyourcode/playground/java/ApplicationTest.java
package com.showmeyourcode.playground.java;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;

class ApplicationTest {

    @Test
    void shouldInitializeClassInstance(){
        assertDoesNotThrow(() ->
                Application.main(new String[0])
        );
    }
}

File: kotlin/src/test/kotlin/com/showmeyourcode/playground/kotlin/ApplicationTest.kt
package com.showmeyourcode.playground.kotlin

import io.kotest.assertions.throwables.shouldNotThrow
import io.kotest.core.spec.style.AnnotationSpec

class ApplicationTest : AnnotationSpec() {
    @Test
    fun `should call the main function without errors`() {
        shouldNotThrow<Throwable> {
            main(arrayOf())
        }
    }
}
